src/DataReader/Cache/CachedCollection.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\DataReader\Cache;
+
+final class CachedCollection
```

Comment:
It looks like it is both internal and redundant. As far as I can see, removing it will make everything a bit simpler.

Reply:
OK. Got the idea.